README.md

```diff
@@ -151,20 +151,23 @@ services:
         #
         # The value 'merge' will ensure that values in this layer specification
         # are merged over existing definitions, whereas 'replace' will entirely
-        # override the existing service spec in the plan with the same name
+        # override the existing service spec in the plan with the same name.
         override: merge | replace
 
+        # (Optional) The command to run the service. This is optional in an
+        # individual layer, but required in the combined layer. The command is
+        # executed directly; use "/bin/sh -c '...'" to run via the shell.
+        #
+        # Example: /usr/bin/somecommand -b -t 30
+        command: <commmand>
```

Comment:
Why did this move?

Reply:
*It* didn't move, I moved it. ;-) I thought it made more sense near the top, right after "override", as it's really the most important field after "override". (More so than summary/description, which are totally optional.)